src/applications/login/containers/MockAuth.jsx

```diff
@@ -4,6 +4,11 @@ import MockAuthButton from '../components/MockAuthButton';
 export default function MockAuth() {
   return (
     <section className="login">
+      <va-banner
+        type="warning"
+        headline="Only for development and localhost environments"
```

Comment:
I think I forgot to update the Figma, my apologies. Can we change this to
`Only available for local and development environments`

Reply:
All good, will do!